wicket-core/src/main/java/org/apache/wicket/ajax/res/js/wicket-ajax-jquery.js

```diff
@@ -36,66 +36,28 @@
 		return;
 	}
 
-	/**
```

Comment:
By the way, just for my education. What's the purpose of the `/*global DOMParser: true, ActiveXObject: true, console: true */` in javascript files?

Reply:
Those are **local** hints for JSHint. The global ones are defined at https://github.com/apache/wicket/blob/886f11de02782feaddc3349e1d7ee599922516c3/testing/wicket-js-tests/Gruntfile.js#L71-L98